### DISCUSSION TITLE
Web 3

### DISCUSSION THREAD

**Discussion by Semnisak** [IBAX-io/go-ibax#468] [Ideas] [answered]
### Clear and concise description of the problem

Web 3 ve bulut teknolojide olmalı

### Suggested solution

Web 3

### Validations

- [X] Read the [docs](https://docs.ibax.io).
- [X] Check that there isn't already an issue that request the same feature to avoid creating a duplicate.

**Comment by ibaxio** [comment, marked as answer]
Elbette, IBAX NETWORK

**Reply by Majezikonline** [reply]
İşte böyle empati yapıp; küçük bir çaba ile google çeviriden dilimize çevirme çabasında bulunan insanlara bayılıyorum. 👍
